### PR TITLE
Remove dconf access and use intltool from shared-modules

### PR DESCRIPTION
**Pull request by bilelmoussaoui**
The old settings can't be migrated as https://gitlab.gnome.org/GNOME/gtranslator/blob/master/data/org.gnome.Gtranslator.gschema.xml.in#L11 uses `/org/gnome/gtranslator/` instead of `/org/gnome/Gtranslator/`

Some of these changes were submitted to the upstream manifest too

### CODE REVIEW

**Comment by TingPing** [comment]
> uses /org/gnome/gtranslator/ instead of /org/gnome/Gtranslator/

It is case insensitive..

File: org.gnome.Gtranslator.yml [hunk 1]
```diff
@@ -21,14 +15,11 @@ cleanup:
   - '*.a'
 finish-args:
   - '--share=ipc'
-  - '--socket=x11'
+  - '--socket=fallback-x11'
```
Comment (TingPing) [inline org.gnome.Gtranslator.yml]: You snuck in a lot of unrelated changes into a single commit which I'm not a fan of.

**Comment by bilelmoussaoui** [comment]
> > uses /org/gnome/gtranslator/ instead of /org/gnome/Gtranslator/
> 
> It is case insensitive..

Well, last time I have tried that with FeedReader it wasn't the case.

File: org.gnome.Gtranslator.yml (hunk 1 above)
Comment (bilelmoussaoui) [reply org.gnome.Gtranslator.yml]: I agree that I should split this a bit

**Comment by TingPing** [comment]
Always has been and it seems like its used: https://github.com/flathub/org.gnome.FeedReader/blob/master/org.gnome.FeedReader.json#L27

If it didn't work flatpak would verbosely warn every start.

**Comment by hfiguiere** [comment]
Runtime is currently 41 and still doesn't do the dconf migration.

Shall this PR simply be updated or just closed?

**Comment by danigm** [comment]
> Runtime is currently 41 and still doesn't do the dconf migration.
> 
> Shall this PR simply be updated or just closed?

This is a bit outdated, maybe it's a good idea to close it and create a [MR upstream](https://gitlab.gnome.org/GNOME/gtranslator/-/blob/master/build-aux/flatpak/org.gnome.Gtranslator.json) with the future changes.

I usually update the flathub yaml with the upstream json manifest, because it's there where I update the dependencies during development.
